frontend/src/scenes/plugins/pluginsLogic.ts

```diff
@@ -112,11 +114,9 @@ export const pluginsLogic = kea<
                     return { ...plugins, [id]: response }
                 },
                 updatePlugin: async ({ id }) => {
-                    const { plugins } = values
-                    const response = await api.update(`api/organizations/@current/plugins/${id}`, {})
+                    const response = await api.create(`api/organizations/@current/plugins/${id}/upgrade`)
```

Comment:
Note: I find the update/upgrade vs patch semantics a bit weird, but not sure how to improve this.

Reply:
(blame Marius) Well initially `PATCH` was responsible for triggering this, but considering `PATCH` is represented as `update` in our API helper and in DRF, it is confusing. Had to change the endpoint to a separate `/upgrade`, but in the UI the feature is still "update".

Reply:
Agreed. This is non-blocking imo.